### ISSUE TITLE
liftOver on .cool files

### ISSUE THREAD

**Issue by tarak77** [open2c/cooler#249]
Hi,
I am trying to use HiCRep(hicreppy in python) to check reproducibility of my contact maps. hicreppy takes .cool files as input. It turns out that one of them was mapped to mm9 and the other was mm10. Is there a way to liftOver the coordinates of mm9 .cool file to mm10??

Any help will be great!

**Comment by nvictus** [comment]
I generally don't recommend trying to lift over Hi-C data. If you had to, it might be sensible to lift over at the pairs level and re-bin (but if you had access to the original reads, it would probably be easier to just re-map).

If you don't have access to the pairs, you're asking for a world of pain. But here's how I might go about it:

1. Dump the pixels with full interval coordinates and counts: (chrom1, start1, end1), (chrom2, start2, end2), count
2. Lift over the axis-1 intervals and the axis-2 intervals separately. Throw away a pixel if either side doesn't lift.
3. The lifted intervals from mm9 are not all going to map cleanly onto regularly binned mm10 intervals, so you'll have to devise an interpolation scheme (the hard part). This could introduce serious artifacts, especially at low resolutions.
4. Aggregate counts from input pixels that ended up mapping to the same output pixel.

tl;dr: don't do it if you don't need to

**Comment by tarak77** [comment]
I see. Yes, I will work with remapping the data.
